### DISCUSSION TITLE
Login Background Image is not working

### DISCUSSION THREAD

**Discussion by alejandrobastos** [requarks/wiki#7235] [Error / Bug Report] [answered]
When setting a Background Image and Applying changes, it doesn't reflect on the Login screen.

Expected behavior:

1. Select a background image on from Assets
2. Insert new image
3. Apply changes
4. See the new image on the login screen

**Comment by NGPixel** [comment, marked as answer]
Make sure the path is available to the Guests group (read:assets permission).

**Reply by alejandrobastos** [reply]
That did the trick! Thanks for the quick response =)
